tests/kit/model_zoo/transformers/chatglm.py

```diff
@@ -0,0 +1,37 @@
+import torch
+import transformers
+
+from ..registry import ModelAttribute, model_zoo
+from .chatglm2_6b.configuration_chatglm import ChatGLMConfig
+from .chatglm2_6b.modeling_chatglm import ChatGLMModel
+
+# ================================
+# Register single-sentence ChatGLM
+# ================================
+
+config = ChatGLMConfig(num_layers=1, padded_vocab_size=65024, hidden_size=64, num_attention_heads=8, rmsnorm=False)
+
+config.__setattr__('original_rope', True)
+config.__setattr__('use_cache', True)
```

Comment:
Why not pass it to the `ChatGLMConfig` directly?

Reply:
they're not included the config class, which means you can't not pass them directly to the ChatGLMConfig, but they will be used when training stage